.circleci/config.yml

```diff
@@ -71,8 +71,8 @@ jobs:
       - run: sh /tmp/get-docker.sh
       - run: |
           cd /tmp
-          wget -nv https://dl.google.com/go/go1.11.11.linux-amd64.tar.gz
-          tar -C /usr/local -xzf go1.11.11.linux-amd64.tar.gz
+          wget -nv https://dl.google.com/go/go1.17.linux-amd64.tar.gz
```

Comment:
lol, 1.17.1 has come out since. it also matches what https://search.nixos.org/packages?channel=unstable&show=go_1_17&from=0&size=50&sort=relevance&type=packages&query=go_1_17 says the package version is.

Reply:
:gotta-gopher-slower-plz: